_posts/2017-01-02-How-to-Write-Assertions-Correctly.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title : "How to Write Assertions Right"
+title : "How to Write Assertions Correctly"
```

Comment:
The reason why I had `right` here is because of the alliteration (i.e. `write` and `right` sounds almost like the same). Is that gramatically incorrect?

Reply:
Although it sounds nice, it isn't really grammatically correct. People will still understand you, but it'll just seem 'off' in when reading it.

Reply:
I see, thank you!